joomla/joomla-cms
# [4.0] CSP vs. Content Security Policy

### Summary of Changes

As done in some places already. Replaces the CSP acronym with Content Security Policy

### Testing Instructions

Check the com_csp component backend views and make sure we use Content Security Policy over CSP

### Expected result

CSP from the title etc should be replaced by Content Security Policy

### Actual result

There are some places were we use CSP and some where we use Content Security Policy

### Documentation Changes Required

none
